config.yaml

```diff
@@ -0,0 +1,107 @@
+default:
+  compat_provisioners:
```

Comment:
`provisioners:`

and then create processors/extension_path.rb as a workaround to set the right path for chef config.

By doing this, https://github.com/teracyhq-incubator/teracy-dev-core/issues/35 is not required.